subsys/bluetooth/controller/ll_sw/nordic/lll/lll_prof.c

```diff
@@ -163,37 +207,39 @@ void lll_prof_send(void)
 	}
 
 	/* generate event if any change */
-	if (chg) {
-		struct node_rx_pdu *rx;
-
-		/* NOTE: enqueue only if rx buffer available, else ignore */
-		rx = ull_pdu_rx_alloc_peek(3);
-		if (rx) {
-			struct pdu_data *pdu;
-			struct profile *p;
-
-			ull_pdu_rx_alloc();
-
-			rx->hdr.type = NODE_RX_TYPE_PROFILE;
-			rx->hdr.handle = 0xFFFF;
-
-			pdu = (void *)rx->pdu;
-			p = &pdu->profile;
-			p->lcur = latency;
-			p->lmin = latency_min;
-			p->lmax = latency_max;
-			p->cur = cputime;
-			p->min = cputime_min;
-			p->max = cputime_max;
-			p->radio = cputime_radio;
-			p->lll = cputime_lll;
-			p->ull_high = cputime_ull_high;
-			p->ull_low = cputime_ull_low;
+	if (!chg) {
+		return -ENODATA;
+	}
 
-			ull_rx_put(rx->hdr.link, rx);
-			ull_rx_sched();
+	/* Allocate if not already allocated */
+	if (!rx) {
+		rx = ull_pdu_rx_alloc();
+		if (!rx) {
+			return -ENOMEM;
 		}
 	}
+
+	/* Generate event with the allocated node rx */
+	rx->hdr.type = NODE_RX_TYPE_PROFILE;
+	rx->hdr.handle = 0xFFFF;
```

Comment:
I feel like I've comment this on some other PR that also used `0xFFFF` as a special handle:

This is a value that probably (definitely) should be declared using a `#define`.

Reply:
Done.